src/components/MarkdownView.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from 'react';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+import rehypeRaw from 'rehype-raw';
+
+interface MarkdownViewProps {
+  url: string;
+}
+
+const MarkdownView = ({ url }: MarkdownViewProps) => {
```

Comment:
Both the header and footer markdown rendering look extremely awful in dark mode. It's rendering black text on a dark background.

Reply:
It still looks bad. Please use the same kind of markdown rendering that is used for protobuf descriptions.